.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,25 @@
+name: "Install Dependencies"
+description: "Install dependencies, fetching from cache when possible"
+inputs:
+  node-version:
+    description: the version of Node.js to install
+    default: 20.18.0
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install pnpm
+      uses: pnpm/action-setup@v4
+      with:
+        version: 9
+
+    - name: Install Node.js ${{ inputs.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ inputs.node-version }}
```

Comment:
shouldn't everywhere use the same version of node? might be worth just hardcoding the version instead of an input

Reply:
1dc6a24